files/angucomplete-alt/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "angucomplete-alt",
+  "repo": "ghiden/angucomplete-alt",
+  "files": {
+    "include": ["./dist/angucomplete-alt.min.js", "angucomplete-alt.js"]
```

Comment:
`angucomplete-alt.js` is not needed. the minified file is enough.
Also change it to `/dist/angucomplete-alt.min.js`